docs/next/modules/en/pages/troubleshooting/troubleshooting.adoc

```diff
@@ -0,0 +1,220 @@
+----
+kubectl krew install lineage
+kubectl krew install get-all
+----
++
+. Then list all existing resources in for example `capi-clusters` namespace where your downstream cluster configuration was deployed.
++
+[source,bash]
+----
+kubectl get-all -n capi-clusters
+----
++
+Example output:
++
+[source,bash]
+----
+NAME                                                                                      NAMESPACE      AGE
+configmap/kube-root-ca.crt                                                                capi-clusters  23h  
+secret/cluster1-shim                                                                     capi-clusters  32s  
+serviceaccount/default                                                                    capi-clusters  23h  
+rke2config.bootstrap.cluster.x-k8s.io/cluster1-mp-0-dm59p                                capi-clusters  32s  
+rke2config.bootstrap.cluster.x-k8s.io/cluster1-mp-1-gv6kh                                capi-clusters  32s  
+rke2configtemplate.bootstrap.cluster.x-k8s.io/cluster1-pool0                             capi-clusters  33s  
+rke2configtemplate.bootstrap.cluster.x-k8s.io/cluster1-pool1                             capi-clusters  33s  
+clusterclass.cluster.x-k8s.io/clusterclass1                                               capi-clusters  33s  
+cluster.cluster.x-k8s.io/cluster1                                                        capi-clusters  32s  
+machinepool.cluster.x-k8s.io/cluster1-mp-0-d4fdv                                         capi-clusters  32s  
+machinepool.cluster.x-k8s.io/cluster1-mp-1-l86kv                                         capi-clusters  32s  
+clustergroup.fleet.cattle.io/clusterclass1                                                capi-clusters  33s  
+azureclusteridentity.infrastructure.cluster.x-k8s.io/cluster-identity                     capi-clusters  33s  
+azuremanagedcluster.infrastructure.cluster.x-k8s.io/cluster1-l2cs6                       capi-clusters  32s  
+azuremanagedclustertemplate.infrastructure.cluster.x-k8s.io/cluster                     capi-clusters  33s  
+azuremanagedcontrolplane.infrastructure.cluster.x-k8s.io/cluster1-rv8v4                  capi-clusters  32s  
+azuremanagedcontrolplanetemplate.infrastructure.cluster.x-k8s.io/cluster1-control-plane  capi-clusters  33s  
+azuremanagedmachinepool.infrastructure.cluster.x-k8s.io/cluster1-mp-0-78tck              capi-clusters  32s  
+azuremanagedmachinepool.infrastructure.cluster.x-k8s.io/cluster1-mp-1-zdscw              capi-clusters  32s  
+azuremanagedmachinepooltemplate.infrastructure.cluster.x-k8s.io/cluster1-pool0           capi-clusters  33s  
+azuremanagedmachinepooltemplate.infrastructure.cluster.x-k8s.io/cluster1-pool1           capi-clusters  33s  
+----
++
+. Then you can check what the relation is between all object resources
++
+[source,bash]
+----
+kubectl lineage -n capi-clusters cluster.cluster.x-k8s.io/cluster1
+----
++
+Output:
++
+image::lineage-cluster-output.png[Output]
```

Comment:
This image did not load for me, I got a 404. Will it be sorted once the page is published?

Reply:
strange it is generating for me